tests/stages.py

```diff
@@ -0,0 +1,322 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+
+import cupy as cp
+import mrc
+import pandas as pd
+
+import cudf
+
+from morpheus._lib.common import FileTypes
+from morpheus.cli.register_stage import register_stage
+from morpheus.config import Config
+from morpheus.io.deserializers import read_file_to_df
+from morpheus.messages import MessageMeta
+from morpheus.messages import MultiMessage
+from morpheus.messages import MultiResponseProbsMessage
+from morpheus.messages import ResponseMemoryProbs
+from morpheus.pipeline.preallocator_mixin import PreallocatorMixin
+from morpheus.pipeline.single_output_source import SingleOutputSource
+from morpheus.pipeline.single_port_stage import SinglePortStage
+from morpheus.pipeline.stream_pair import StreamPair
+from morpheus.utils import compare_df
+from morpheus.utils.atomic_integer import AtomicInteger
+
+
+@register_stage("unittest-conv-msg")
+class ConvMsg(SinglePortStage):
```

Comment:
Move all of these stages into a subpackage. Add `stages/__init__.py` and then add each stage to a separate file.